### ISSUE TITLE
[EN][NPC][Firelands] Alysrazor

### ISSUE THREAD

**Issue by sunshitwowsucks** [Atlantiss/BugTracker#8096]
We could not complete Firelands with the new update because we wiped to Alysrazor when Majordomo summoned her. And when the feather was there to star the encounter, we just got the movement speed buff..

I believe this happened after the latest update on 8/25/16

Edit: Now the feather comes back after a few minutes, and when you click it, Nothing happens.

**Comment by Wolffenstein** [comment]
This would mean you already did Alysrazor on this ID, there is no other option.

**Comment by sunshitwowsucks** [comment]
We did not kill Alys yet.

**Comment by sanhechuan** [comment]
I tried continuing one run yesterday and that is before the server kept crashing. We got Beth+Rhyo down and wiped on Alys,  so we went to Bale+Shannox, after wiping on Alys over and over,  we finally downed it before the server crashed for the last time.  I wasn't going to put up with more crashes, so we decided then called it and told them we'd continue it tomorrow, and now there's no gate and I cannot summon the gate (no portal either), and for some reason there's an Alys feather like we wiped, but it gives us the ability to move while casting + movement speed.
